prisma-fmt/tests/references/scenarios/model_as_type/result.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "uri": "file:///path/to/a.prisma",
+    "range": {
+      "start": {
+        "line": 10,
+        "character": 6
+      },
+      "end": {
+        "line": 10,
+        "character": 13
+      }
+    }
+  },
+  {
+    "uri": "file:///path/to/b.prisma",
+    "range": {
+      "start": {
+        "line": 3,
+        "character": 11
+      },
+      "end": {
+        "line": 3,
+        "character": 18
+      }
+    }
```

Comment:
Is it expected that the line is always 1 lower than the actual reference?

Example: `range.start.line = 10` for `a.prisma`, although `UserTwo` is line 11

Reply:
Yes, this is the difference in indexing from 0 vs 1. Lines start from 1 in the editor but start from 0 in the actual params 😅

Reply:
Makes sense 👍